common/lib/xmodule/xmodule/js/fixtures/problem_content.html

```diff
@@ -16,6 +16,10 @@ <h2 class="problem-header">Problem Header</h2>
     <button class="reset">Reset</button>
     <button class="save">Save</button>
     <button class="show"><span class="show-label">Show Answer(s)</span> <span class="sr">(for question(s) above - adjacent to each field)</span></button>
+    <div class="reset-error">
+      <i class="fa fa-exclamation-triangle"></i>
+      <span class="reset-error">${_('You have to make an attempt before you reset the problem')}</span><span class="sr">${_('You have to make an attempt before you reset the problem')}</span>
```

Comment:
@mehrozezahid We don't need to apply the `reset-error` class again here. We have same message in both the span tags. Remove the second span and `<span class="sr">Warning:</span>` before first span. Finally it will looks like

```HTML
<span class="sr">Warning:</span>
<span class="reset-error">${_('You have to make an attempt before you reset the problem')}</span>
```

Also shouldn't we render the message received from server instead of hardcoding the message here. In this way others can use element.